src/odemis/acq/align/delphi_man_calib.py

```diff
@@ -262,45 +263,47 @@ def man_calib(logpath):
                   "                    rotation: " + str(irot) + "\n"
                   "                    scale-xy: " + str(iscale_xy) + "\n"
                   "                    shear: " + str(ishear))
-        print_col(ANSI_YELLOW,
+        print_col(ANSI_YELLOW, -
```

Comment:
what does the - do here?

Reply:
It makes sure that Python cannot parse this file :-P

Thanks for catching this. Just pushed a fixed version.

Reply:
haha here I was, thinking it was some fancy trick :)